requirements/pip.txt

```diff
@@ -101,11 +101,19 @@ django-cors-middleware==1.4.0  # pyup: ignore
 # User agent parsing - used for analytics purposes
 user-agents==2.2.0
 
+
 # Utilities used to upload build media to cloud storage
 # django-storages is pinned to this particular commit because it
 # supports generating URLs with other method than GET when using
-# private buckets
-git+https://github.com/jschneier/django-storages@d0f027c98a877f75615cfc42b4d51c038fa41bf6#egg=django-storages[azure]==1.9.1
+# private buckets.
+#
+# Besides, support for the corresponding AWS_BUCKET_ACL and
+# AWS_AUTO_CREATE_BUCKET settings have been removed in 1.10. We depend on this
+# in our Docker setup. We can upgrade it but we need to add a
+# `create_buckets.sh` to be called on `--init` as we used to do for Azurite
+# https://github.com/jschneier/django-storages/pull/636
+git+https://github.com/jschneier/django-storages@d0f027c98a877f75615cfc42b4d51c038fa41bf6#egg=django-storages[boto3]==1.9.1
```

Comment:
We can track this with a separate issue perhaps, we probably don't want to get too far behind on django-storages releases

Reply:
I'm not sure to follow you here.

We are not changing the version in this PR. I just changed `azure` by `boto3` + add a comment explaining _why_ we can't upgrade.